core/shared/src/test/scala/com/monovore/decline/HelpSpec.scala

```diff
@@ -16,16 +17,18 @@ class HelpSpec extends WordSpec with Matchers {
       ) {
         val first = Opts.flag("first", short = "F", help = "First option.").orFalse
         val second = Opts.option[Long]("second", help = "Second option.").orNone
+        val third = Opts.option[Long]("third", help = "Third option.") orElse
+          Opts.envVar[Long]("THIRD")
         val subcommands =
           Opts.subcommand("run", "Run a task?") {
             Opts.argument[String]("task")
           }
 
-        (first, second, subcommands).tupled
+        (first, second, third, subcommands).tupled
       }
 
       Help.fromCommand(parser).toString should equal(
-        """Usage: program [--first] [--second <integer>] run
+        """Usage: program [--first] [--second <integer>] --third <integer> run
```

Comment:
I think this is not the rendering we want: this usage statement makes it looks like `--third` is required, but it's legal to call this program without passing `--third`.

I think changing the added line in `Usage` to match the `Pure` case instead of the `Missing` case should do the right thing, but I haven't tested!